SharedPackages/BrowserServicesKit/Sources/Subscription/Managers/SubscriptionManagerV2.swift

```diff
@@ -63,6 +63,7 @@ public enum SubscriptionPixelType {
     case migrationSucceeded
     case migrationFailed(Error)
     case subscriptionIsActive
+    case subscriptionExpired
```

Comment:
Also this pixel is only added to V2, I think until we are 100% released we should probably add it to both. If we think it would be useful we can optionally add auth version as a param, or maybe as a separate pixel.